src/Robo/Commands/Tests/AllCommand.php

```diff
@@ -20,6 +20,7 @@ public function tests() {
     $this->invokeCommands([
       'tests:behat:run',
       'tests:phpunit:run',
+      'tests:drupal:run',
```

Comment:
This new command should be used Drupal tests, whether through `phpunit` or `run-tests.sh`